lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -6702,7 +6708,42 @@ namespace Js
         Assert(element1 != NULL);
         Assert(element2 != NULL);
 
+#ifndef DBG
```

Comment:
>DBG [](start = 8, length = 3)

why not do in the enable_debug_config_option?

Reply:
I've changed it to this.